Sources/WalletConnectVerify/OriginVerifier.swift

```diff
@@ -5,22 +5,36 @@ public final class OriginVerifier {
         case registrationFailed
     }
     
-    private let verifyHost: String
+    private var verifyHost: String
     
     init(verifyHost: String) {
         self.verifyHost = verifyHost
     }
     
     func verifyOrigin(assertionId: String) async throws -> String {
-        let httpClient = HTTPNetworkClient(host: verifyHost)
-        let response = try await httpClient.request(
-            VerifyResponse.self,
-            at: VerifyAPI.resolve(assertionId: assertionId)
-        )
-        guard let origin = response.origin else {
-            throw Errors.registrationFailed
+        let sessionConfiguration = URLSessionConfiguration.default
+        sessionConfiguration.timeoutIntervalForRequest = 5.0
```

Comment:
5.0 seconds is intended for fallback correct?
what if it's already on .org, do we need a timeout then?